packages/api/src/@types/ScdlDataIntegration.ts

```diff
@@ -0,0 +1,15 @@
+type BaseScdlParseArgs = [producerSlug: string, exportDate: string];
+export type ScdlParseArgs = BaseScdlParseArgs & [delimiter?: string, quote?: string];
+export type ScdlParseXlsArgs = BaseScdlParseArgs & [pageName?: string, rowOffset?: number | string];
```

Comment:
```suggestion
type BaseScdlParseArgs = {producerSlug: string, exportDate: string};
export type ScdlParseArgs = BaseScdlParseArgs & {delimiter?: string, quote?: string};
export type ScdlParseXlsArgs = BaseScdlParseArgs & {pageName?: string, rowOffset?: number | string};
```
J'ai l'impression que c'est cette modif que @mxmeunier voudrait et dont il parle plus bas. J'ai tendance à être d'accord ça portera moins à confusion sur l'ordre à respecter

Reply:
oui carrément